### PR TITLE
:running: Additional e2e test fixes

### PR DESCRIPTION
**Pull request by detiber**
**What this PR does / why we need it**:
Additional e2e fixes, with these changes the e2e tests are running locally for me again.

### CODE REVIEW

File: test/e2e/e2e_suite_test.go [hunk 1]
```diff
@@ -176,6 +173,26 @@ func applyManifests(kindCluster kind.Cluster, manifests *string) {
...
+		}
+	}
+	Expect(err).NotTo(HaveOccurred())
+
+	// write out the manifests
+	manifestFile := path.Join(suiteTmpDir, "cluster-api-components.yaml")
+	Expect(ioutil.WriteFile(manifestFile, capiManifests, 0644)).To(Succeed())
+
+	// apply generated manifests
+	applyManifests(kindCluster, &manifestFile)
```
Comment (ncdc) [inline test/e2e/e2e_suite_test.go]: Follow-up: change `applyManifests` so it takes in a `string`, not a `*string`?

File: test/e2e/e2e_suite_test.go [hunk 2]
```diff
@@ -176,6 +173,26 @@ func applyManifests(kindCluster kind.Cluster, manifests *string) {
 	kindCluster.ApplyYAML(*manifests)
 }
 
+func deployCAPIComponents(kindCluster kind.Cluster) {
+	fmt.Fprintf(GinkgoWriter, "Generating CAPI manifests\n")
+
+	// Build the manifests using kustomize
+	capiManifests, err := exec.Command(*kustomizeBinary, "build", "https://github.com/kubernetes-sigs/cluster-api//config/default").Output()
```
Comment (ncdc) [inline test/e2e/e2e_suite_test.go]: Any concern about reading all of this into memory vs using `StdoutPipe` and an `io.Copy`?

**Comment by ncdc** [comment]
No blocking comments - will lgtm after we wrap up discussing the exec.Command. Thanks!

File: test/e2e/e2e_suite_test.go (hunk 2 above)
Comment (detiber) [reply test/e2e/e2e_suite_test.go]: Maybe, but this is only temporary until we publish capi manifests

File: test/e2e/e2e_suite_test.go (hunk 1 above)
Comment (detiber) [reply test/e2e/e2e_suite_test.go]: hmm, now that you mention it this should probably just call `kindCluster.ApplyYAML` instead :joy:

**Pull request merged**